products/mattermost.md

```diff
@@ -16,6 +16,12 @@ auto:
 
 # EOL date can be found on https://docs.mattermost.com/upgrade/release-lifecycle.html
 releases:
+-   releaseCycle: "9.5"
+    releaseDate: 2024-02-16
+    eol: 2024-11-15
+    latest: '9.5.0'
+    latestReleaseDate: 2024-02-16
```

Comment:
```suggestion
    latest: '9.5.1'
    latestReleaseDate: 2024-02-16
```

Will be updated to 2024-02-09 by automation, but nothing much we can do for now.